jwql/utils/constants.py

```diff
@@ -106,6 +98,27 @@
 JWST_MAST_SERVICES = ['Mast.Jwst.Filtered.{}'.format(value.title()) for value in
                       JWST_INSTRUMENT_NAMES]
 
+ANOMALIES_PER_INST = {
+    # anomalies affecting all instruments:
+    'snowball': JWST_INSTRUMENT_NAMES,
+    'cosmic_ray_shower': JWST_INSTRUMENT_NAMES,
+    'crosstalk': JWST_INSTRUMENT_NAMES,
+    'data_transfer_error': JWST_INSTRUMENT_NAMES,
+    'diffraction_spike': JWST_INSTRUMENT_NAMES,
+    'excessive_saturation': JWST_INSTRUMENT_NAMES,
+    'ghost': JWST_INSTRUMENT_NAMES,
+    'guidestar_failure': JWST_INSTRUMENT_NAMES,
+    'persistence': JWST_INSTRUMENT_NAMES,
+    'satellite_trail': JWST_INSTRUMENT_NAMES,
+    # instrument-specific anomalies:
+    'optical_short': ['nirspec'],  # Only for MOS observations
+    'dragons_breath': ['nircam'],
+    # additional anomalies:
+    'other': JWST_INSTRUMENT_NAMES}
+
+# Defines the possible anomalies (with rendered name) to flag through the web app
+ANOMALY_CHOICES = [(anomaly, inflection.titleize(anomaly)) for anomaly in ANOMALIES_PER_INST]
```

Comment:
@tnking97 Could you put `ANOMALIES_PER_INST` and `ANOMALY_CHOICES` towards the top of this file, in alphabetical order with the rest of the variables?

Reply:
Sure! Since JWST_INSTRUMENT_NAMES isn't defined at the top of this file, I replaced JWST_INSTRUMENT_NAMES with ['fgs', 'miri', 'nircam', 'niriss', 'nirspec'] in the ANOMALIES_PER_INSTRUMENT variable.